### ISSUE TITLE
 protocol version not supported.

### ISSUE THREAD

**Issue by chenyg0911** [goharbor/harbor-helm#18]
I'm newbie to harbor. I've installed the harbor follow the readme. I use the default values.yaml which is ingress enable.After installation, all seams work fine. all pods running.
But, I try to open https://core.harbor.domain. I got the error:the  protocol version not supported.
some problem about the https.
when I set:
`
  tls:
    enabled: false
`
I can access UI thrugh the http://core.harbor.domain.

How should I do?

**Comment by chenyg0911** [comment]
nginx-ingress version: nginx-ingress-controller   0.15.0

**Comment by ywk253100** [comment]
I think maybe there is something wrong when negotiating with HTTP2, the Nginx ingress controller enables HTTP2 by default. Is there any detail information?

**Comment by chenyg0911** [comment]
it seam the nginx-ingress only support SSLv2, edit the configmap, change to:sl_protocols SSLv3 TLSv1 TLSv1.1 TLSv1.2.  But, I got a error: invalid certificate ...
Does any body can give me a guide?

**Comment by chenyg0911** [comment]
Since chart auto generate ca, how to import the ca files in browser and docker client?

**Comment by ywk253100** [comment]
You can get the ca file in `Administration->Configuration->System Settings` by login with admin user.

**Comment by chenyg0911** [comment]
will try it later. reinstall the chart. It seem some problem with UI images. always got a 404.

**Comment by joostcoelingh** [comment]
I can confirm @chenyg0911 's issue, the new chart gives a 404 after the deployment (worked fine last week).

browser shows:
```
404 Page Not Found
Home
```
Logs don't give much more info either:
```
[harbor-harbor-adminserver-77664dd55b-6jn9s] 10.233.125.25 - - [10/Sep/2018:10:39:20 +0000] "GET /api/configurations HTTP/1.1" 200 1732 
[harbor-harbor-ui-86c5597d9b-7dd8d] 2018-09-10T10:39:20Z [DEBUG] [security.go:290]: user information is nil 
[harbor-harbor-ui-86c5597d9b-7dd8d] 2018-09-10T10:39:20Z [DEBUG] [security.go:304]: using local database project manager 
[harbor-harbor-ui-86c5597d9b-7dd8d] 2018-09-10T10:39:20Z [DEBUG] [security.go:306]: creating local database security context... 
[harbor-harbor-ui-86c5597d9b-7dd8d] 2018-09-10T10:39:21Z [DEBUG] [security.go:233]: can not get user information from session 
```

**Comment by wody** [comment]
The 404 problem is related to #23

**Comment by ywk253100** [comment]
@chenyg0911 The 404 error will be fixed by #26, after it merged you can try again, thanks.

**Comment by chenyg0911** [comment]
OK. Will try it later. thanks!

Wenkai Yin <notifications@github.com> 于2018年9月12日周三 上午10:02写道：

**Comment by chenyg0911** [comment]
now the ui work.  when I visit:https://core.harbor.domain. I got a ca error in browser. 
`
SEC_ERROR_REUSED_ISSUER_AND_SERIAL 
`


kubectl -n ingress-nginx exec -it ingress-nginx-controller-n2x5m -- cat /etc/nginx/nginx.conf
`
... 
	## start server core.harbor.domain
	server {
		server_name core.harbor.domain ;
		
		listen 80;
		
		listen [::]:80;
		
		set $proxy_upstream_name "-";
		
		listen 443  ssl http2;
		
		listen [::]:443  ssl http2;
		
		# PEM sha: 8ebc32bf369782ef91a9de184a02f558be104a26
		ssl_certificate                         /etc/ingress-controller/ssl/default-my-release-harbor-ingress.pem;
		ssl_certificate_key                     /etc/ingress-controller/ssl/default-my-release-harbor-ingress.pem;
		
		ssl_trusted_certificate                 /etc/ingress-controller/ssl/default-my-release-harbor-ingress-full-chain.pem;
		ssl_stapling                            on;
		ssl_stapling_verify                     on;
...

`

Should I  import ca for brower first?  and which ca I should import?
chart values.yaml
`
 tls:
    enabled: true
    # Fill the secretName if you want to use the certificate of 
    # yourself when Harbor serves with HTTPS. A certificate will 
    # be generated automatically by the chart if leave it empty
    secretName: ""

**Comment by ywk253100** [comment]
Are you using Firefox or Safari? There is a known issue https://github.com/goharbor/harbor/issues/5711. Try the Chrome instead.

**Comment by ywk253100** [comment]
BTW, how do you fix the `the protocol version not supported` error?

**Comment by chenyg0911** [comment]
Yes, reconfig ingress to add tls2.0 and tls 1.1, the `the protocol version not supported error` is fix. But I still can't use https,whatever chrome or firefox. 
attachment is chrome (Ver 69) clipper. I must unsecure http to visit UI.
![2018-09-18 09-56-13](https://user-images.githubusercontent.com/16010372/45659973-69ed1b00-bb29-11e8-9dcd-3ae5228304fc.png)

**Comment by ywk253100** [comment]
Seems something is wrong with the certificate. Can you check the certificate by clicking the "不安全" in the left of browser address bar?

**Comment by chenyg0911** [comment]
Yes, The problem is cert. But I don't konw which cert file I should import.
![2018-09-18 10-54-04](https://user-images.githubusercontent.com/16010372/45661743-6cec0980-bb31-11e8-984b-1b23d3f9d7ba.png)

**Comment by ywk253100** [comment]
Check the detail of the certificate by clicking `证书`.
